From: Zhao Gongyi <zhaogongyi@huawei.com>
Subject: selftests/vm: add local_config.h and local_config.mk to .gitignore
Date: Thu, 3 Nov 2022 08:57:54 +0800

Add local_config.h and local_config.mk to .gitignore to avoid accidentally
checking it in.

Link: https://lkml.kernel.org/r/20221103005754.205420-1-zhaogongyi@huawei.com
Signed-off-by: Zhao Gongyi <zhaogongyi@huawei.com>
Cc: Shuah Khan <shuah@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/testing/selftests/vm/.gitignore |    2 ++
 1 file changed, 2 insertions(+)

--- a/tools/testing/selftests/vm/.gitignore~selftests-vm-add-local_configh-and-local_configmk-to-gitignore
+++ a/tools/testing/selftests/vm/.gitignore
@@ -33,3 +33,5 @@ memfd_secret
 soft-dirty
 split_huge_page_test
 ksm_tests
+local_config.h
+local_config.mk
_
